packages/zapp/console/src/components/Literals/Scalar/StructuredDatasetValue.tsx

```diff
@@ -0,0 +1,122 @@
+import {
+  SimpleType,
+  StructuredDataset,
+  StructuredDatasetColumn,
+  StructuredDatasetColumnType,
+} from 'models/Common/types';
+import * as React from 'react';
+import { PrintList } from '../PrintList';
+import { PrintValue } from '../PrintValue';
+import { useLiteralStyles } from '../styles';
+import { ValueLabel } from '../ValueLabel';
+
+function getSimpleTypeDisplayValue(type: SimpleType) {
```

Comment:
nit: Maybe we can implement it similar to 
```
export const taskTypeToNodeExecutionDisplayType: {..}
```
then usage would be `simpleTypeDisplayValue(type) ?? unknown;`

Totally your call, I just not a big fun of such switches.

Reply:
One more way to do it create a string file and use it from there (similar to   tableLabel_name: 'execution id',):
```
const str = {
  simpleType_: 'unknown',
  simpleType_0: 'none',
  simpleType_1: 'integer'
  ...
}
```
and than call the string as ` t(patternKey('simpleType', type))`
I don't think current implementation will resolve unsupported type to simpleType_ , so additional simpleType_unknown string could be added instead, with type ?? 'unknown' check in t() call...

p.S> Not sure this is a good choice for this function, but for one below - it could be a perfect chioce.